### DISCUSSION TITLE
q4mvvuXqKdAJPtOTyvYYNvPlA3BHY3X6eG1MzldiYiFoSxzaWRTmgtzhPIZ09IGI

### DISCUSSION THREAD

**Discussion by ZerkyLiu** [Meloong-Git/PCL#2319] [讨论]
q4mvvuXqKdAJPtOTyvYYNvPlA3BHY3X6eG1MzldiYiFoSxzaWRTmgtzhPIZ09IGI
hsmq6zL+QKXHB1tWMM5Znv1aYd4HnY7CW6olF4yh9n7qz699mMAKTvC66GD+ToHMePpB9HN+4dQvaIgzIIEZkf32QQnZXhEG

**Comment by WForst-Breeze** [comment]
这里是 PCL2 仓库，不是您的记事本.jpg🤔

**Reply by iPodtouch6557** [reply]
这就像抖音有人把别人的私信当自己的记事本😆

**Comment by ZerkyLiu** [comment]
sorry啊这其实是用PCL加密的文本（以后在不这么完了）
![image](https://github.com/Hex-Dragon/PCL2/assets/115980764/07981fde-8c5d-4ad9-9717-df614210fb29)

**Reply by NebulaTechs** [reply]
这是什么

**Comment by iPodtouch6557** [comment]
~好嘛以后国防部交流就用这个了~
另外好像该功能已被拒绝……但是搜不到了

**Reply by NebulaTechs** [reply]
我想知道，这是什么加密

**Reply by iPodtouch6557** [reply]
……翻译过来是：能不能让PCL同时启动多个MC版本啊！现在还需要等待一个启动完了才能启动另一个
大概就这意思

**Reply by NebulaTechs** [reply]
，。我只是想知道这是什么加密算法
